particles/MeshParticles.cpp

```diff
@@ -2,11 +2,8 @@
 
 namespace particles
 {
```

Comment:
I refactored TexturedMeshForParticles to move the mesh data outside of it ; now it will only be about the mesh specification (except for the size thing, which  I believe could be made better but I have no idea of a good design for it right now)